### ISSUE TITLE
Update search views translations

### ISSUE THREAD

**Issue by lgontard** [ytgov/yukon-ca#787]
A few of the views that list documents and searches show English (view) content.

https://yukon.ca/fr/documents

> Sorry, no results were found. Try eliminating one of the filters applied.

https://yukon.ca/fr/arts-et-culture/activites

> No events found. Try removing or changing a filter.

We have the translations. Just need to find them.

<img width="536" alt="image" src="https://github.com/user-attachments/assets/fde8425e-657b-45e9-9a81-0cf7066c40ae" />
<img width="512" alt="image" src="https://github.com/user-attachments/assets/a818a150-5bc4-4247-aaa9-db2e6a314fc8" />

**Comment by w3creatives** [comment]
Please follow the steps below:

https://yukon.ca/fr/admin/structure/views/view/documents_browse/translate

Please the screenshot below and add translation in the right side input box. Then save.
https://prnt.sc/8KG_kgk0aOi0

https://yukon.ca/fr/admin/structure/views/view/events_listing/translate/fr/edit

Please the screenshot below and add translation in the right side input box. Then save.
https://prnt.sc/ja5rCI5MhANc

**Comment by lgontard** [comment]
Updated this https://yukon.ca/fr/documents "Sorry, no results were found. Try eliminating one of the filters applied." 
Waiting on translation for other one.

**Comment by lgontard** [comment]
Replaced the English on Events. This is resolved.

**Comment by leeomara** [comment]
I'd like to make sure the updated config for these changes are captured in code.

**Comment by w3creatives** [comment]
Please review this PR - #824 

This is a config change.

**Comment by lgontard** [comment]
Here's the translation:

Sorry, no results were found. Try eliminating one of the filters you applied.


Désolé, aucun résultat n’a été trouvé. Essayez de retirer un filtre.

**Comment by w3creatives** [comment]
It is already done. Screenshot attached.

![Image](https://github.com/user-attachments/assets/fe5127bf-96ce-461e-ad19-c92bcb5a63ce)
